src/renderers/dom/stack/client/ReactDOMComponent.js

```diff
@@ -1005,10 +1002,11 @@ ReactDOMComponent.Mixin = {
         }
       } else if (isCustomComponentTag) {
         if (!RESERVED_PROPS.hasOwnProperty(propKey)) {
-          DOMPropertyOperations.setValueForAttribute(
+          DOMPropertyOperations.setValueForProperty(
```

Comment:
Same here - React props are now dom element properties, not attributes. The `true` once again indicates that this is a custom element node.